cves/kernel/CVE-2020-14386.yml

```diff
@@ -456,7 +501,10 @@ mistakes:
 
     Write a thoughtful entry here that people in the software engineering
     industry would find interesting.
-  answer:
+  answer: 
+    I think the mistake was a slip and/or a panning error. Previous commits
+    added in checks to prevent buffer overflows for other variables in the
```

Comment:
Which commits? Possibly add these to the interesting commits section showing that some buffer overflows were caught and fixed while others went unnoticed.

Reply:
added